.github/workflows/release.yml

```diff
@@ -9,6 +9,11 @@ on:
       - "!release-manual"
       - "!release-manual-*"
 
+permissions: 
+  contents: write
+  pull-requests: write
```

Comment:
Do you have links to what these are doing versus what the default permissions are?  Is this more restrictive?  Or just more explicit?